docs/application-connector/docs/041-environment-mapping-custom-resource.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EnvironmentMapping
+type: Custom Resource
+---
+
+The `environmentmappings.remoteenvironment.kyma.cx` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to enable the Remote Environment with the name corresponding to the EnvironmentMapping in a given Namespace. To get the up-to-date CRD and show the output in the yaml format, run this command:
```

Comment:
Please mention at the end, that EM enables API as service classes from RE in given namespace.

Reply:
but only if you are using ns-scoped ServiceBroker. If RemoteEnivironmentBroker is registered as cluster-wide then all RE classes are always visible